test/locust-users.py

```diff
@@ -0,0 +1,46 @@
+
+# Copyright Contributors to the Open Cluster Management project
+from locust import HttpUser, task, between, TaskSet
+import json
+import urllib3
+urllib3.disable_warnings() # Suppress warning from unverified TLS connection (verify=false)
+
+userCount = 0
+token_string = "Replace with <oc whoami -t>"
+
+class UserBehavior(TaskSet):
+
+    @task
+    def do_search(self):
+        f = open("search-query-template.json",)
+        j = json.load(f)
+        # 
+        # TODO: Here we can manipulate j.variables to pass different input parameters to the search query.
+        # 
+        self.client.payload = j
+        self.client.headers
+        self.do_post()
+        print("Sent search for user {}".format(self.user.name))        
+
+    def do_post(self):
+        self.client.post("/searchapi/graphql", 
+            headers={"Authorization": "Bearer " + token_string},
+            json=self.client.payload, verify=False)
+
```

Comment:
@jlpadilla do you think it would be helpful to get:

1. time taken to get response
2. the http return code

And put them into a CSV file - so that we can get statistics?

Reply:
Locust is doing these 2 things automatically for us. :-)